tests/pylorax/test_projects.py

```diff
@@ -330,7 +330,7 @@ def singlerepo_v0():
         "check_gpg": True,
         "check_ssl": True,
         "gpgkey_urls": [
-            "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-32-x86_64"
```

Comment:
This will break running things in podman. Ideally the cockpit tests for master branch should be running against a rawhide image, but I realize that's hard to do. The test setup should probably make the change, possibly based on the TEST_OS value?

Reply:
Right, there is no Rawhide image available in Cockpit CI but there is `fedora-testing` (I don't know which version this is). Whatever we decide to do Dockerfile.test (Travis CI) and the new scenario added by Jakub must use the same Fedora version. Otherwise we'll start seeing differences. 

One option, which I don't like, is to have this switch merged only for RHEL branches and leave master as-is.

Reply:
I changed the way `TEST_OS` variable is defined in the Makefile and changed this test to use it. I don't know if it's the most elegant way but it works. I'll be glad to hear any suggestions to improve this.